earth_enterprise/rpms/build.gradle

```diff
@@ -402,16 +402,64 @@ if (project.hasProperty('buildOpenGee')) {
     openGeeCommonRpm.dependsOn stageOpenGeeInstall
 }
 
+task openGeeCommonDeb (type: GeeDeb, dependsOn: openGeeSharedFiles) {
+    packageName = openGeeCommonRpm.packageName
+    release = openGeeCommonRpm.release
+    version = openGeeVersion
+    user = openGeeCommonRpm.user
+    permissionGroup = openGeeCommonRpm.permissionGroup
+    packageGroup = 'misc'
+    formatPackageDescription(openGeeCommonRpm.summary + '\n' + openGeeCommonRpm.packageDescription)
+    license = openGeeCommonRpm.license
+    priority = 'optional'
+
+    setArch('amd64')	// if this is not specified, the architecture is x86_64, and dpkg won't install without the --force-architecture flag
+
+    requires('flex')
+    requires('freeglut3-dev')
```

Comment:
Do you know if this is a runtime requirement? `freeglut3` without "-dev" might work fine. We can either test that now or add it to a follow-up ticket

Reply:
By "test that now", do you mean just the build and install? I assume there's nothing in opengee-common that we can test by executing, correct?

Reply:
It builds and installs fine with `freeglut3` instead of `freeglut3-dev`. I guess that means it'll be fine, but I don't know. Do you want me to add that to this pull request?

Reply:
That's true, we won't be able to truly test until we do the follow-up. For now, I say let's go with the non-dev version. Fewer packages is better, and `freeglut3-dev` installs about 5 more packages than just plain `freeglut3`.

Reply:
Sounds like a plan. I'll get it added shortly.